cli/src/pcluster/constants.py

```diff
@@ -206,3 +207,37 @@
 
 IAM_NAME_PREFIX_LENGTH_LIMIT = 30
 IAM_PATH_LENGTH_LIMIT = 512
+
+
+# Operations support
+# By default, all operations are considered supported.
+# To mark an operation as unsupported for certain regions,
+# add the entry to the map below by region prefixes.
+class Operation(Enum):
+    """
+    Enumeration of operations.
+
+    We do not expect this enumeration to list all the operations,
+    but at least those that are considered for operations flagging.
+    """
+
+    BUILD_IMAGE = "build-image"
+    DELETE_IMAGE = "delete-image"
+    DESCRIBE_IMAGE = "describe-image"
+    LIST_IMAGES = "list-images"
+    EXPORT_IMAGE_LOGS = "export-image-logs"
+    GET_IMAGE_LOG_EVENTS = "get-image-log-events"
+    GET_IMAGE_STACK_EVENTS = "get-image-stack-events"
+    LIST_IMAGE_LOG_STREAMS = "list-image-log-streams"
```

Comment:
Should we add all the operations here?
Is there a way to check if we're missing some operations from the API? (e.g. with a unit test?)

Reply:
> Should we add all the operations here?

We can for completeness, but it's not strictly required. 
An entry here is required only if we are going to set it as unsupported in some regions.
I agree on adding all of theme for completeness.

> Is there a way to check if we're missing some operations from the API? (e.g. with a unit test?)

Not now. To do so we must:
1. add all the operations into the enumeration, and
2. extend the current unit test to cover them;

I will do.